src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todos } from './types/Todos'
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todos: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [titleEntered, setTitleEntered] = useState(true);
+  const [userChosen, setUserChosen] = useState(true);
+
+  const nextTodoId = Math.max(...visibleTodos.map(todo => todo.id)) + 1;
```

Comment:
move it to the separate function and call this function where you create new todo